internal/server/data/organization_test.go

```diff
@@ -13,60 +13,58 @@ import (
 // PostgreSQL only has microsecond precision
 var cmpTimeWithDBPrecision = cmpopts.EquateApproxTime(time.Microsecond)
 
-func TestCreateOrganizationAndSetContext(t *testing.T) {
-	pgsql := postgresDriver(t)
-	db := setupDB(t, pgsql)
+func TestCreateOrganization(t *testing.T) {
+	runDBTests(t, func(t *testing.T, db *DB) {
```

Comment:
Updated this test to use `runDBTests` like all the others. We only really need to test against postgres right now, but I expect `runDBTests` to be useful in the future despite that.

One use case that comes to mind is testing postgres upgrades. We can run all of our `data` tests against multiple versions of postgres to catch problems.